### ISSUE TITLE
Having issues downloading content pages from http://de.metapedia.org/

### ISSUE THREAD

**Issue by meezy23** [WikiTeam/wikiteam#276]
Hello!

I recently installed the dumpgenerator and it worked fine for getting various wikis. Now I tried to get a big dump from the (abstrusely racist) wiki http://de.metapedia.org/ which is crucial for my future work, but it doesn't work as expected: I just get the namespaces that aren't the actual articles but just discussion sites, categories, user sites etc.
I tried both getting the full dump and restricting the downloads to namespaces with the id 0, but both attempts failed. Am I doing something wrong or are the people from Metapedia somehow actively defending the download of the articles? If so, why can I get them with `curl` (also, why does it work for all the other namespaces).

I added the files that I got with `python dumpgenerator.py --index=http://de.metapedia.org/index.php --xml --curonly` excluding the HTML files and the large XML: [demetapediaorg-20160615-wikidump.zip](https://github.com/WikiTeam/wikiteam/files/316230/demetapediaorg-20160615-wikidump.zip)

Any hint would be appreciated, I really don't feel like writing a crawler myself with my shitty perl skills.

Using Mac OS btw.

Thanks in advance,
M.

**Comment by nemobis** [comment]
I note the API is disabled. http://de.metapedia.org/m/api.php

**Comment by Hydriz** [comment]
Can't seem to reproduce this for other sites (like http://cs.metapedia.org which also has API disabled). Might be a site-specific configuration that is blocking our crawler/

**Comment by nemobis** [comment]
I don't see anything special in http://de.metapedia.org/wiki/Spezial:AllPages nor http://de.metapedia.org/wiki/Spezial:Export/Main_page .

**Comment by meezy23** [comment]
> Can't seem to reproduce this for other sites
Yeah me neither, that's why I was inclined to post here in the first place.. What I don't understand is that it works for the non-articles?! I checked the website source code for the varying namespaces, there don't seem to be any big differences in the encoding of the namespace section between e.g. a [portal](http://de.metapedia.org/wiki/Portal:Film) or an [article](http://de.metapedia.org/wiki/Film).
